### PR TITLE
Microsoft excel: new phrasing Q114 + 2 new questions (with answers)

### PR DESCRIPTION
**Pull request by Knackii**
No, last choice of Q115 is not a typo from me.

### CODE REVIEW

File: microsoft-excel/microsoft-excel-quiz.md [hunk 1]
```diff
@@ -1040,3 +1040,19 @@ https://support.microsoft.com/en-us/office/print-headings-or-titles-on-every-pag
...
+##### Q114. What is the output of the formula =(8+2*3)/7?
+
+- [ ] 13
+- [x] 7
+- [ ] 11
+- [ ] 15
+
+#### Q115. The amount of sales tax on each sale is calculated as the selling price times the quantity sold times the sales tax rate. What formula would you use in celle E4 to then be able ro copy that cell to the rest of the column?
+
+(missing screenshot)
```
Comment (sathir) [inline microsoft-excel/microsoft-excel-quiz.md:1053]: Better to add the screenshot

**Review by sathir — CHANGES_REQUESTED** [review]

File: microsoft-excel/microsoft-excel-quiz.md [hunk 2]
```diff
@@ -1040,3 +1040,19 @@ https://support.microsoft.com/en-us/office/print-headings-or-titles-on-every-pag
 - [ ] `=CLEAN(TRIM(A2))`
 - [ ] `=CHOOSE(A2)`
 - [ ] `=TRIM(A2)`
+
+##### Q114. What is the output of the formula =(8+2*3)/7?
+
```
[the remark this answers is not in the document]
Comment (Knackii) [reply microsoft-excel/microsoft-excel-quiz.md:1045]: It's litteraly just "=(8+2*3)/7"

File: microsoft-excel/microsoft-excel-quiz.md (hunk 1 above)
Comment (Knackii) [reply microsoft-excel/microsoft-excel-quiz.md:1053]: I don't have it

**Review by Ebazhanov — APPROVED** [review]
Thank you @Knackii for your great input! 🌹

**Comment by Ebazhanov** [comment]
@all-contributors please add @Knackii for design, code

**Pull request merged**